### PR TITLE
Update plugin description

### PR DESCRIPTION
**Pull request by aslo**
Improves the description that is shown on the Jetbrains repository website and in the IDEA update plugin screen.

### CODE REVIEW

**Comment by aslo** [comment]
@etanshaul I took a stab at writing the description. Let me know what you think.

File: google-cloud-tools-plugin/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -17,8 +17,17 @@
...
   <id>com.google.gct.core</id>
   <name>Google Cloud Tools</name>
-  <description>Debug Java applications running in the Google cloud from inside IDEA.
-Code inspections for App Engine Java code.</description>
+  <description>
+    <![CDATA[<html>
+      <p>Provides support for developing Google App Engine applications. Features include:</p>
+      <ul>
+        <li>Debug Java applications running in the Google cloud from inside IDEA.</li>
+        <li>Code inspections for App Engine Java code.</li>
```
Comment (etanshaul) [inline google-cloud-tools-plugin/resources/META-INF/plugin.xml]: .. App Engine standard environment ..

File: google-cloud-tools-plugin/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -17,8 +17,17 @@
...
+        <li>Running applications on a local development server.</li>
```
Comment (etanshaul) [inline google-cloud-tools-plugin/resources/META-INF/plugin.xml]: Run & Debug applications on a ..

File: google-cloud-tools-plugin/resources/META-INF/plugin.xml [hunk 3]
```diff
@@ -17,8 +17,17 @@
...
+        <li>Deployment to App Engine's standard and flexible environments.</li>
```
Comment (etanshaul) [inline google-cloud-tools-plugin/resources/META-INF/plugin.xml]: Deployment -> Deploy
.. to the App Engine standard and flexible environments.

**Comment by etanshaul** [comment]
Added some comments. I think this is a good as far as content. Maybe @patflynn has any suggestions for changes / missed features to include here?

**Comment by etanshaul** [comment]
Also are you sure this section can take HTML?

**Comment by aslo** [comment]
Yep - my quick tests (building the plugin and installing from disk) show that the html is rendered properly, same as in the change-notes section.

**Pull request merged**